tensorflow/core/kernels/mkl_fused_batch_norm_op.cc

```diff
@@ -544,6 +545,7 @@ class MklFusedBatchNormGradOp : public OpKernel {
     }
 
     void MklCreateInputLayout(OpKernelContext* context) {
+      const Tensor& input = MklGetInput(context, 0);
```

Comment:
This local variable also isn't used anywhere in this function.

Reply:
Good catch!  

Yes, this line of code should be there. And it was in a function with MKL-ML integration which will be removed in the long run since we have MKL-DNN integration.

I will clean up this code, after all primitive reuse PRs have been done (only Relu one remains). I have a TODO list based on Rasmus's suggestions, which were applied only to individual PRs  (thus
only on related changed files), and will have a "clean-up" PR. 

Thanks!